renderers/server-render.jsx

```diff
@@ -0,0 +1,172 @@
+import Rx from 'rxjs';
```

Comment:
the server entry point(s) are in the individual apps, which have `src/server.js` or `src/server-locale` as the entry point for the locale bundles and `scripts/start.js` or whatever for the app server bundle.

`server-render.jsx` and `browser-render.jsx` just provide rendering functions to the app's entry points